portfolio/src/main/webapp/script.js

```diff
@@ -53,4 +53,20 @@ function deleteComment(comment) {
   const params = new URLSearchParams();
   params.append('id', comment.id);
   fetch('/delete-comment', {method: 'POST', body: params});
+}
+
+function translateComment() {
+    // getting comment and language code
```

Comment:
Usually don't need comments like this since I can get this information reading the next 2 lines. This is because you chose good variable names :).